### DISCUSSION TITLE
Welcome to vue-cal V4 Discussions!

### DISCUSSION THREAD

**Discussion by antoniandre** [antoniandre/vue-cal-v4#589] [Announcements]
## 👋 Welcome to V4!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

The new discussion channel is open for everybody who want to discuss usage of Vue Cal that is not directly an issue. Like questions.

**Comment by antoniandre** [comment]
Please acknowledge that the V5 is in the pipe (if you're interested check this repo https://github.com/antoniandre/vue-cal) and I'm actively working on it. 💪
It will be better, faster stronger :p and trying to be more intuitive in the communication with Vue Cal.
thanks for you support all!  🚀💚🚀💚🚀💚🚀💚🚀💚

**Reply by jonastahl** [reply]
Will it have Typescript support? :)

**Reply by subdee** [reply]
Sounds great! I hope there will be some upgrade documentation coming with it from v4.

**Reply by antoniandre** [reply]
@SecretJ12 not from the start, but most likely added later. You're also most welcome to contribute type definitions :)
@subdee, Yes I will add a migration guide.

**Reply by jonastahl** [reply]
I can certainly give it a try. Could you ping me when you reach a more or less stable version worth typing?

**Reply by antoniandre** [reply]
For sure, we have a plan! @SecretJ12

**Reply by antoniandre** [reply]
Hey @jonastahl!
I think we have reached a more or less stable state and you can already start implementing some TS types! :)
That would be just amazing for all the TS community.  🙌 
The project is still evolving and some features are still missing but the core is really there and stable 💥 
please check the codebase in this repo https://github.com/antoniandre/vue-cal in the main branch and let me know if you've got questions! 
let's goooo 🚀

**Reply by jonastahl** [reply]
Hi @antoniandre,

thanks for pinging me, sounds amazing :)
I will definitely look into this and try to create the types.
Currently I'm quite stacked with lectures and private appointments so please don't expect the results to early. I'm gonna reach out to you!

**Reply by antoniandre** [reply]
let's continue the v5 discussion at https://github.com/antoniandre/vue-cal/discussions/5! :)
